packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -104,7 +141,50 @@ export const TranslationSidebar = ({
                 </div>
               )}
             </Tab.Panel>
-            <Tab.Panel>{t('common:coming_soon')}</Tab.Panel>
+            <Tab.Panel>
+              {/* TODO: add author and timestamp */}
+              <div className="flex flex-col gap-2 pb-2">
```

Comment:
A couple high level UI design thoughts here for discussion. These are my opinions, so I'm open to feedback.

* I think we can condense the design a little bit and communicate the same information. Something like this:

**Translator Notes**
*Updated at 1/2/24 12:30 pm by Adrian Rocke*

* When we want to permit editing, I would prefer to just show the rich text editor and automatically save changes after debouncing. This prevents changes from being lost because somebody didn't press save, and reduces the needed button presses. This works because the rich text input is WYSIWYG where editing and preview are the same thing.
* Finally, we also need to disable editing if the user isn't a language translator. If that's the case, then we should just show the note rather than a rich text editor.

Reply:
This makes sense 👍🏽

Reply:
I will experiment with this later on in the week.